### ISSUE TITLE
Searching before exhausting queue

### ISSUE THREAD

**Issue by jsmith613** [raulrene/twitter-contest-js-bot#33]
The code definitely seems to make requests before the queue has been fully exhausted.  When run, the queue is continually repopulated after attempting to retweet.  I will look into the issue and see if I can fix.

**Comment by calvinti12** [comment]
Have you found a fix for this issue yet I’m having the same issue

**Comment by jsmith613** [comment]
No, I haven't really had the chance now that my school has started back up.  Just by running it a few times, it has worked sometimes.  It seems like the problem might be when it comes across tweets it has already seen.  I know if it sees a tweet it adds it to a blocked list.  Could be something wrong there but I'm not sure.

**Comment by calvinti12** [comment]
OK keep in mind I’m willing to pay to help get it up and running
